source/beamline/REIXS/REIXSXESMCPDetector.cpp

```diff
@@ -343,3 +371,13 @@ void REIXSXESMCPDetector::acquisitionCancelledHelper(){
 
 	emit imageDataChanged();
 }
+
+void REIXSXESMCPDetector::toggleVeto(bool on)
+{
+	if (vetoControl_->isConnected() && vetoStateControl_->isConnected()) {
+		if (on)
+			vetoControl_->move(0); // veto off is to enable updating
+		else
+			vetoControl_->move(1); // veto on is to disable updating
+	}
+}
```

Comment:
@iainworkman yes, the function name is not clear. It might be clear to rename the `toggleVetor()` function to `toggleMCPCountUpdate()`.